SwiftWisdomTests/StandardLibrary/Numbers/IntTests.swift

```diff
@@ -1,19 +0,0 @@
-//
-//  IntTests.swift
-//  SwiftWisdom
-//
-//  Created by Logan Wright on 3/31/16.
-//  Copyright © 2016 Intrepid. All rights reserved.
-//
-
-import XCTest
-
-class IntTests: XCTestCase {
-    func testOddEven() {
```

Comment:
This file only tests this one method, which is already tested in `IntegerTypeTests.swift`. I didn't see any real distinction between the two tests, so I figured it was an organizational accident.